### PR TITLE
Renaming files

### PR DESCRIPTION
**Pull request by anucreative**

### CODE REVIEW

File: User Flow Docs.sketchplugin/Contents/Sketch/populate-user-flows.js [hunk 1]
```diff
@@ -85,6 +85,8 @@ function populateScreens(path, templatePage, numberOfScreensPerPage, projectName
     	var key = folder.lastPathComponent();
     	var folderData = screenData[ key ];
 
+    	log( folderData.title )
```
Comment (matt-oakes) [inline User Flow Docs.sketchplugin/Contents/Sketch/populate-user-flows.js]: Is it worth taking this log out?

File: User Flow Docs.sketchplugin/Contents/Sketch/export-for-user-flows.js [hunk 2]
```diff
@@ -66,8 +68,13 @@ function exportForUserFlows( context ) {
                 // Check that it's an artboard and not a slice
                 if ( artboard.class() == MSArtboardGroup.class() ) {
                     // Export the artboard to the correct file location
-                    var artboardPath = pageDirectoryUrl.URLByAppendingPathComponent( artboard.name() + fileExtension ).path()
-                    [doc saveArtboardOrSlice:artboard toFile:artboardPath];
+                    var artboardPath = pageDirectoryUrl.URLByAppendingPathComponent( artboard.name() + FILE_EXTENSION ).path()
+
+                    var sizes = artboard.exportOptions().sizes().array()
+                    var slices = MSSliceMaker.slicesFromExportableLayer_sizes(artboard, sizes).objectEnumerator()
```
Comment (matt-oakes) [inline User Flow Docs.sketchplugin/Contents/Sketch/export-for-user-flows.js:74]: Spaces around paramaters

File: User Flow Docs.sketchplugin/Contents/Sketch/export-for-user-flows.js [hunk 3]
```diff
@@ -66,8 +68,13 @@ function exportForUserFlows( context ) {
...
+                    while( slice = slices.nextObject() ) {
+                        doc.saveArtboardOrSlice_toFile(slice, artboardPath);
```
Comment (matt-oakes) [inline User Flow Docs.sketchplugin/Contents/Sketch/export-for-user-flows.js:76]: Spaces around paramaters

File: User Flow Docs.sketchplugin/Contents/Sketch/export-for-user-flows.js [hunk 4]
```diff
@@ -84,18 +91,18 @@ function exportForUserFlows( context ) {
         }
 
         // Add the sections
-        jsonObject[ sectionsJSONKey ] = sectionsJson;
+        jsonObject[ SECTIONS_JSON_KEY ] = sectionsJson;
 
         // Ask the user for the name of the project
         var projectName = [doc askForUserInput:"What is this project called?" initialValue:guessProjectName()]
-        jsonObject[projectNameJSONKey] = new String(projectName)
+        jsonObject[PROJECT_NAME_JSON_KEY] = new String(projectName)
```
Comment (matto1990) [inline User Flow Docs.sketchplugin/Contents/Sketch/export-for-user-flows.js]: Spaces

**Comment by matt-oakes** [comment]
I was going to add comments to each line, but it looks like you've got a mixed code style. There is both code like:

```
hello(thingy);
```

and

```
hello( thingy );
```

I think it needs to be one or the other :+1:

**Pull request merged**